settings.gradle.kts

```diff
@@ -1,2 +1 @@
-rootProject.buildFileName = "build.gradle.kts"
```

Comment:
Why isn't this needed?

Reply:
No need to decleared explicitly, gradle can auto load `build.gradle.kts`s